### DISCUSSION TITLE
background items added

### DISCUSSION THREAD

**Discussion by Leoknowstheway** [SpotX-Official/SpotX-Bash#16] [Q&A] [answered]
<img width="348" alt="Screenshot 2023-07-23 at 9 43 37 PM" src="https://github.com/jetfir3/SpotX-Bash/assets/87140438/afa56133-9e3c-4179-a70f-0efe8ddd6996">

I just restarted my macbook and I've been spammed by "Background Items Added" how do I stop this? It's been going on for more than an hour

**Comment by jetfir3** [comment, marked as answer]
This seems to be an issue between macOS Ventura(+) and apps that have background tasks... in this case, Spotify. 

The problem is outlined in this discussion - https://discussions.apple.com/thread/254571804

The last comment in the thread seems to work for most - _As far as Spotify goes, change Spotify's setting "Startup and window behaviour" to "No"._

Regardless, this issue is unrelated to `SpotX` or `SpotX-Bash`.
